# Service to set a runtime parameter

# The setting index to set to
uint32 setting_ind

---
